src/components/views/dialogs/BaseDialog.js

```diff
@@ -69,17 +83,17 @@ export default React.createClass({
         const TintableSvg = sdk.getComponent("elements.TintableSvg");
 
         return (
-            <div onKeyDown={this._onKeyDown} className={this.props.className}>
+            <FocusTrap onKeyDown={this._onKeyDown} className={this.props.className} role="dialog" aria-labelledby='mx_BaseDialog_title' aria-describedby={this.props.contentId}>
```

Comment:
Long line split in ab0ff9b7
As for the possible component ID clashes I guess Modal only allows single dialog to be rendered into the DOM so at the DOM level this should hopefully not be an issue. If I have missed something and in deed this might be dangerous or inappropriate in other way then I can append some randomly generated number at the end of these IDs. Should I do that?

Reply:
Sorry, to be clear I think that we can ignore this on exactly the basis you mentioned - Modal only allows a single dialog to be rendered.